src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java

```diff
@@ -0,0 +1,54 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+import java.time.Instant;
+
+/**
+ * Entity class for money wallet.
+ *
+ * @author DawidStuzynski
+ * @version 1.0
+ * @since track-expenses-app-backend 2.0
+ */
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class Wallet implements Serializable {
+
+    /**
+     * Class version 1.0 -> 100010L
+     */
```

Comment:
Could you please explain what do you mean in this Javadoc by Class version 1.0 -> 100010L?